### ISSUE TITLE
Wrong tray icon color for xfce4-power-manager

### ISSUE THREAD

**Issue by aacebedo** [vinceliuice/vimix-gtk-themes#204]
Distribution: Endeavour OS (arch based)
Desktop environment: Xfce + i3
Installing method: aur/vimix-gtk-themes-git r358.8b76023-1
App name: xfce4-power-manager 4.16.0

I use the `vimix-dark-laptop-ruby` theme and the `PapirusDark` theme.
The icon of the tray despite being white on my disk is recolored to a dark color, making it invisible.
I checked with strace `xfce4-power-manager` that the icon loaded is correct `/usr/share/icons/Papirus-Dark/symbolic/status/battery-level-80-symbolic.svg` but it is displayed in a dark color instead of white:
![xfce4-power-manager](https://user-images.githubusercontent.com/1217680/112753574-477cc800-8fd8-11eb-811b-d303d29365ba.png)

The only way to fix this is to add
* {color:#FFFFFF;} to ~/.local/gtk-3.0/gtk.css however it will have an impact on other places than this specific icon.
Is there another way to solve this ?

**Comment by prakash2033** [comment]
@aacebedo have you found any better solution yet?

**Comment by aacebedo** [comment]
Nope. I switched to sway and nixos since and I use another bar so did not see the issue again.

**Comment by prakash2033** [comment]
I’m on awesome wm and your solution does fix the issue but then it affects my thunderbird message pane and may be some other app who knows. 

Googling for hours to get the right css name just for xfce4-power-manager icon is not helping. In some places people suggested 
#xfce4-power-manager-plugin image {}

That is also not working.

Thanks for your reply anyway.
